consumers/aws.go

```diff
@@ -53,6 +53,7 @@ func withClient(c AWSClient, groupID string) *awsConsumer {
 func (a *awsConsumer) Sync(endpoints []*pkg.Endpoint) error {
 	kubeRecords, err := a.endpointsToRecords(endpoints)
 	if err != nil {
+		log.Errorf("failed to convert endpoints to RRS: %v. Aborting sync...", err)
 		return err
```

Comment:
yes, totally make sense. Initially I wanted to print debug message, but I thought debug is not relevant here either. If error is not logged here with additional message, then it is a bit difficult to figure where the flow is aborted